src/utils/helpers.ts

```diff
@@ -82,10 +82,11 @@ export const pluck = (obj: { [key: string]: any }, keys: string[]) => {
 
 export const isBoolean = (v: any): v is boolean => typeof v === 'boolean';
 export const isDefined = (v: any) => v !== null && v !== undefined;
-export const isUndefined = (v: any) => v === null || v === undefined;
+export const isUndefined = (v: any): v is null | undefined => v === null || v === undefined;
 export const isFunction = (v: any): v is Function => typeof v === 'function';
 export const isNumber = (v: any): v is boolean => typeof v === 'number';
```

Comment:
Not related to this PR, but I think there's a tiny typo in this guard:
```suggestion
export const isNumber = (v: any): v is number => typeof v === 'number';
```

Reply:
😱